android/libraries/rib-base/src/main/java/com/badoo/ribs/android/ActivityStarter.kt

```diff
@@ -13,29 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.uber.rib.core;
+package com.badoo.ribs.android
```

Comment:
We still have old `com.uber.rib.core.ActivityStarter` in rib-android-test.

Reply:
Will fix in #10